README.md

```diff
@@ -94,11 +94,14 @@ If you want to generate binaries for different platforms:
 make compile
 ```
 ### Manual start
-Set some values
-+ set the value `secretNameDB` in the file `helm-charts/app-to-monitor/values.yaml` to be the same as the value of `secretName` in the file `helm-charts/database-to-integrity-sum/values.yaml`
-+ set the value `releaseNameDB` in the file `helm-charts/app-to-monitor/values.yaml` to be the same as the release name you will use when installing the database
+Set some environment variables to configure DB params:
+- PGPASSWORD
```

Comment:
Setting `PGPASSWORD` and other DB envs,  you can just export `DB_PASSWORD`

Reply:
It's just a reminder for a user.